app/assets/javascripts/controllers/container_dashboard/container_dashboard_controller.js

```diff
@@ -0,0 +1,118 @@
+                count: 0
+            };
+
+            $scope.updateStatus = function (statusObject, data) {
+                statusObject.status = [];
+                if (data) {
+                    statusObject.count = data.count;
+                    if (data.errorCount > 0) {
+                        statusObject.status.push(
+                            {
+                                iconClass: "pficon-error-circle-o",
+                                count: data.errorCount
+                            }
+                        );
+                    }
+                    if (data.warningCount > 0) {
+                        statusObject.status.push(
+                            {
+                                iconClass: "pficon-warning-triangle-o",
+                                count: data.warningCount
+                            }
+                        );
+                    }
+                } else {
+                    statusObject.count = 0;
+                }
+            };
+
+            $scope.refresh = function() {
+                var id;
+                if ($location.absUrl().match("show/$") || $location.absUrl().match("show$")) {
+                    id = '';
+                }
+                else {
+                    id = '/'+ (/container_dashboard\/show\/(\d+)/.exec($location.absUrl())[1]);
+                }
+
+                var url = '/container_dashboard/data'+id;
+                $http.get(url).success(function(response) {
+                    'use strict';
+
+                    var data = response.data;
+                    var providers = data.providers;
+                    if (providers)
+                    {
+                        $scope.providers.count = providers.length;
+                        providers.forEach(function (item) {
+                            $scope.providers.notifications.push({
+                                iconClass: item.iconClass,
+                                count: item.count
+                                //href: "#containers/providers/?filter=" + item.providerType
+                            })
+                        });
+                    }
+
+                    $scope.updateStatus($scope.nodesStatus,      data.status.nodes);
+                    $scope.updateStatus($scope.containersStatus, data.status.containers);
+                    $scope.updateStatus($scope.registriesStatus, data.status.registries);
+                    $scope.updateStatus($scope.projectsStatus,   data.status.projects);
+                    $scope.updateStatus($scope.podsStatus,       data.status.pods);
+                    $scope.updateStatus($scope.servicesStatus,   data.status.services);
+                    $scope.updateStatus($scope.imagesStatus,     data.status.images);
+                    $scope.updateStatus($scope.routesStatus,     data.status.routes);
+                });
+        };
+        $scope.refresh(); // reload every interval?
```

Comment:
please check the "auto refresh" for topology with interval, it should be similar here